data/language/en-GB.txt

```diff
@@ -3642,6 +3642,7 @@ STR_6383    :Open download page
 STR_6384    :Snow
 STR_6385    :Heavy Snow
 STR_6386    :Blizzard
+STR_6387    :Brake/Boost speed
```

Comment:
The Tile Inspector uses colons after headers, i.e.:
```suggestion
STR_6387    :Brake/Boost speed:
```

Reply:
No need to amend and force push after every change, btw. We can squash everything nicely after reviewing.